app/facades/food_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodFacade
+  class << self
+    def food_search(search_params)
+      results = FoodService.food_search(search_params)[:foods]
+      food = results.map do |info|
+        Food.new(info)
+      end
+      food[0..9]
+    end
```

Comment:
I believe there's a way to limit the responses on your api call through a passing a certain parameter, you may consider doing that instead of selecting the first 10. Essentially, you're making food objects out of ALL data that's passed back, and then only selecting 10. This isn't very efficient if you truly only want 10 to get sent back to the view.

Reply:
In here is also where you could send a 2d array to get stored into the respective variables I mentioned in the controller. [results[:totalHits],food]